python_bot.py

```diff
@@ -558,7 +570,7 @@ def emojify(string):
   elif(message.content.startswith('!ping')):
     await ping_command(message)
   elif(message.content.startswith('!stopvoice')):
-    if(message.author.id == '159785058381725696' or message.author.id == '328175857707253760'):
+    if(message.author.id == id_branden or message.author.id == '328175857707253760'):
```

Comment:
nit: If you like you can remove this unknown user id. It's someone's for the intern discord server, where this bot originated. Or from my EQ2 guild discord. Either way, whoever owns this ID won't be using it.

Reply:
Did you miss this one?

Reply:
yep, I was half awake when I was fiddling around this morning.